react-front-end/tsrc/components/Lightbox.tsx

```diff
@@ -135,7 +128,9 @@ const {
   openSummaryPage: openSummaryPageString,
 } = languageStrings.lightboxComponent;
 
-const { close: labelClose, openInNewWindow: labelOpenInNewWindow } =
+const { copy: labelCopy } = languageStrings.embedCode;
```

Comment:
Probably more meaningful to go with:

```suggestion
const { copy: labelCopyEmbedCode } = languageStrings.embedCode;
```